Snowflake.Data/Core/SFStatement.cs

```diff
@@ -17,6 +17,61 @@
 
 namespace Snowflake.Data.Core
 {
+    /// <summary>

```

Comment:
let's remove unused imports from this file

Reply:
Removed

Reply:
what about `using System.ComponentModel;`?

Reply:
Removed as well. Sorry, I posted that before replacing the `Description` attribute